synapse/federation/sender/transaction_manager.py

```diff
@@ -104,13 +104,13 @@ async def send_new_transaction(
                 len(edus),
             )
 
-            transaction = Transaction.create_new(
+            transaction = Transaction(
                 origin_server_ts=int(self.clock.time_msec()),
                 transaction_id=txn_id,
                 origin=self._server_name,
                 destination=destination,
-                pdus=pdus,
-                edus=edus,
+                pdus=[p.get_pdu_json() for p in pdus],
```

Comment:
This will interfere with magic below where we serialise the events *just* before sending out the request, this is for the `age` stuff.

Reply:
I'm not sure I follow, this moved logic out of `create_new` into the caller, so it should be identical.

Reply:
Oh, hmm, you're right. I'm just confused then. Ignore me. Nothing to see here. Lalalalala.